src/ui/LikeButton/index.svelte

```diff
@@ -21,6 +21,11 @@
   function startVote(e: MouseEvent) {
     if (disabled) return
 
+    // Disable right click event
+    if (e.which === 3 || e.button === 2) {
```

Comment:
`e.which` is a non-standart property, hence it is redundant.
```suggestion
    // Checking right button mouse click
    if (e.button === 2) {
```